src/solvers/dgmulti/dg.jl

```diff
@@ -353,9 +349,10 @@ end
 
 
 # Todo: DGMulti. Specialize for modal DG on curved meshes using WADG
-function invert_jacobian!(du, mesh::DGMultiMesh, equations, dg::DGMulti, cache)
+# inverts Jacobian and scales by -1.0
+function invert_jacobian!(du, mesh::DGMultiMesh, equations, dg::DGMulti, cache; scaling=-1.0)
```

Comment:
```suggestion
function invert_jacobian!(du, mesh::DGMultiMesh, equations, dg::DGMulti, cache; scaling=-1)
```
In case we want to use `Float32` etc.